db/migrate/20221214090559_add_devise_to_users.rb

```diff
@@ -0,0 +1,48 @@
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email, null: false
+      t.string :encrypted_password, null: false
+
+      ## Recoverable
+      t.string :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
+
+      ## Lockable
+      # t.integer  :failed_attempts, default: 0, null: false # Only if lock strategy is :failed_attempts
+      # t.string   :unlock_token # Only if unlock strategy is :email or :both
+      # t.datetime :locked_at
+
+      # Uncomment below if timestamps were not included in your original model.
+      # t.timestamps null: false
+    end
+
+    add_index :users, :email, unique: true
+    add_index :users, :reset_password_token, unique: true
+    # add_index :users, :confirmation_token,   unique: true
```

Comment:
- [ ] To enable confirmation email as I mentioned above you'll need to uncomment this line as well and then redo your migration.

Reply:
**Dear @mirouhml , I tried what you said , but it does not allow user to log in ! however it registered before. So I decided to remove that confirmable to let the app work**
![confirm](https://user-images.githubusercontent.com/35001742/207930872-53f4a9ad-f5b9-43f9-b881-1a1b0a4a271d.JPG)